.github/workflows/build.yml

```diff
@@ -374,13 +374,12 @@ jobs:
           timestamp-digest: SHA256
           timeout: 600
 
-      - name: "[Windows 11 ARM64] Install WiX Toolset3"
-        if: matrix.os == 'windows-11-arm'
+      - name: "[Windows] Install WiX Toolset"
+        if: runner.os == 'Windows'
+        shell: pwsh
         run: |
-          choco install wixtoolset
-          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
-          refreshenv
-          echo "WIX=$env:WIX" >> $env:GITHUB_ENV
+          dotnet.exe tool install --global wix --version 6.0.1
+          wix extension add --global WixToolset.UI.wixext/6.0.1
```

Comment:
How has the situation with the WIX environment variable changed?

Reply:
It is not needed anymore for WiX 4. As CI passes and generates proper MSI files, we can be sure, that the executable was found.